shell/browser/api/electron_api_session.cc

```diff
@@ -1688,8 +1689,17 @@ gin::WeakCell<Session>* Session::FromBrowserContext(
 }
 
 // static
-Session* Session::CreateFrom(v8::Isolate* isolate,
-                             ElectronBrowserContext* browser_context) {
+Session* Session::FromOrCreate(v8::Isolate* isolate,
+                               content::BrowserContext* context) {
+  if (ElectronBrowserContext::IsValidContext(context))
+    return FromOrCreate(isolate, static_cast<ElectronBrowserContext*>(context));
+  DCHECK(false);
+  return {};
```

Comment:
Expecting a DCHECK that is always false but returning a null value in release builds seems to contradict, would this be better with `NOTREACHED` ?

Reply:
I'm fine with making this `NOTREACHED`